### PR TITLE
Content

### PR DESCRIPTION
**Pull request by jaymahima**

### CODE REVIEW

File: kANe/v1p1/01_meaning_of_dharma.md [hunk 1]
```diff
@@ -3,15 +3,16 @@ title = "01 Meaning of Dharma"
...
 
 The present work will deal with the sources of _dharma_, their contents, their chronology and other kindred matters. As the material is vast and the number of works is extremely large, only a few selected works and some important authors will be taken up for detailed treatment. More space will be devoted to comparatively early works.
 
-[^1]: पि॒तुं नु स्तो॑षं म॒हो ध॒र्माणं॒ तवि॑षीम् । This occurs in शुक्लयजुर्वेद 34.7.
+[^1]: पि॒तुं नु स्तोषं म॒हो ध॒र्माणं॒ तवि॑षीम् । This occurs in शुक्लयजुर्वेद 34.7.
```
Comment (vvasuki) [inline kANe/v1p1/01_meaning_of_dharma.md:13]: अत्र स्वरं यथापूर्वं करोतु

File: kANe/v1p1/01_meaning_of_dharma.md [hunk 2]
```diff
@@ -3,15 +3,16 @@ title = "01 Meaning of Dharma"
...
 
-[^2]: इ॒मम॑ञ्ज॒स्पामु॒भये॑ अकृण्वत ध॒र्माण॑म॒ग्निं वि॒दथ॑स्य॒ साध॑नम् ।
+[^2]: इ॒मम॑ञ्ज॒स्पामु॒भये अकृण्वत ध॒र्माण॑म॒ग्निं वि॒दथ॑स्य॒ साध॑नम् ।
```
Comment (vvasuki) [inline kANe/v1p1/01_meaning_of_dharma.md:15]: अत्र स्वरं यथापूर्वं करोतु

File: kANe/v1p1/01_meaning_of_dharma.md [hunk 3]
```diff
@@ -25,13 +26,13 @@ The present work will deal with the sources of _dharma_, their contents, their c
...
+[^12]: हरदत्त on गौ. ध. सू. 19. 1 and गोविन्दराज on मनु 2.25 give the same fivefold classification. The Bhaviṣyapurāṇa (in Brāhmaparva 181, 10-15) sets out this fivefold dharma (Venk, Press ed.). The Gaut. Dh. 19. 1 says 'Ukto varṇadharmaścāśramadharmśca' and then proceeds to dilate upon prāyaścittas (i.e. naimittika dharma). The Āp. Dh. S. begins, 'atbātaḥ Sāmayācārikān _dharmān_ vyākhyāsyāmaḥ' (We shall expound the dharmas relating to the practices based on convention and adds immediately that the conventions settled by those who are conversant with dharma and the authorities and not any and every convention whatever,' The Āp. Gṛ. S. begins 'Atha karmāṇyācarād-yāni gṛhyante.' That is actions are Śrutilakṣaṇa and ācāralakṣaṇa. Samaya has several meanings as Amara says 'Samayāḥ śapathācārakālasiddhānta-saṁvidaḥ.'
```
Comment (vvasuki) [inline kANe/v1p1/01_meaning_of_dharma.md:35]: No objection to using either ṃ or ṁ ([IAST](https://en.wikipedia.org/wiki/IAST) vs [ISO](https://en.wikipedia.org/wiki/ISO_15919)). परिवर्तने श्रमो मास्तु।

**Review by vvasuki — CHANGES_REQUESTED** [review]
केचन दोषा अत्र सूचिताः। परिहृत्य, नुदित्वा (push कृत्वा), https://github.com/vishvAsa/kalpAntaram/pull/193 इत्य् उद्घाट्य "समीकृतम्" इति प्रतिसन्देशम् उररीकरोतु ।

**Comment by jaymahima** [comment]
क्षम्यताम् महोदय। अहं अवगन्तुं प्रयते। archive मध्ये स्तोषं इति अस्ति। स्तौषं इत्यस्य परिष्कारं कृतम्। तद् न साधु वा? 193 कथम् उद्घाट्यते?

धन्यवाद:

From: Vishvas Vasuki विश्वासः ***@***.***>
Date: Saturday, November 19, 2022 at 8:52 PM
To: vishvAsa/kalpAntaram ***@***.***>
Cc: jaymahima ***@***.***>, Author ***@***.***>
Subject: Re: [vishvAsa/kalpAntaram] Content (PR #193)

@vvasuki requested changes on this pull request.

केचन दोषा अत्र सूचिताः। परिहृत्य, नुदित्वा (push कृत्वा), #193<https://github.com/vishvAsa/kalpAntaram/pull/193> इत्य् उद्घाट्य "समीकृतम्" इति प्रतिसन्देशम् उररीकरोतु ।

________________________________

In kANe/v1p1/01_meaning_of_dharma.md<https://github.com/vishvAsa/kalpAntaram/pull/193#discussion_r1027171231>:

>

 The present work will deal with the sources of _dharma_, their contents, their chronology and other kindred matters. As the material is vast and the number of works is extremely large, only a few selected works and some important authors will be taken up for detailed treatment. More space will be devoted to comparatively early works.



-[^1]: पि॒तुं नु स्तो॑षं म॒हो ध॒र्माणं॒ तवि॑षीम् । This occurs in शुक्लयजुर्वेद 34.7.

+[^1]: पि॒तुं नु स्तोषं म॒हो ध॒र्माणं॒ तवि॑षीम् । This occurs in शुक्लयजुर्वेद 34.7.

अत्र स्वरं यथापूर्वं करोतु

________________________________

In kANe/v1p1/01_meaning_of_dharma.md<https://github.com/vishvAsa/kalpAntaram/pull/193#discussion_r1027171234>:

>

-[^2]: इ॒मम॑ञ्ज॒स्पामु॒भये॑ अकृण्वत ध॒र्माण॑म॒ग्निं वि॒दथ॑स्य॒ साध॑नम् ।

+[^2]: इ॒मम॑ञ्ज॒स्पामु॒भये अकृण्वत ध॒र्माण॑म॒ग्निं वि॒दथ॑स्य॒ साध॑नम् ।

अत्र स्वरं यथापूर्वं करोतु

________________________________

In kANe/v1p1/01_meaning_of_dharma.md<https://github.com/vishvAsa/kalpAntaram/pull/193#discussion_r1027171467>:

>

 [^10]: त्रयो धर्मस्कन्धा यज्ञोध्ययनं दानमिति प्रथमस्तप एवेति द्वितीयो ब्रह्मचार्याचार्यकुलवासी तृतीयोत्यन्तमात्मानमाचार्यकुलेवसादयन् सर्व एते पुण्यलोका भवन्ति ब्रह्मसंस्थोमृतत्वमेति ॥ vide वेदान्तसूत्र III. 4. 18-20 for a discussion of this passage.



 [^11]: 'सर्वधर्मसूत्राणां वर्णाश्रमधर्मोपदेशित्वात्' p. 237.



-[^12]: हरदत्त on गौ. ध. सू. 19. 1 and गोविन्दराज on मनु 2.25 give the same fivefold classification. The Bhaviṣyapurāṇa (in Brāhmaparva 181, 10-15) sets out this fivefold dharma (Venk, Press ed.). The Gaut. Dh. 19. 1 says 'Ukto varṇadharmaścāśramadharmśca' and then proceeds to dilate upon prāyaścittas (i.e. naimittika dharma). The Āp. Dh. S. begins, 'atbātaḥ Sāmayācārikān _dharmān_ vyākhyāsyāmaḥ' (We shall expound the dharmas relating to the practices based on convention and adds immediately that the conventions settled by those who are conversant with dharma and the authorities and not any and every convention whatever,' The Āp. Gṛ. S. begins 'Atha karmāṇyācarād-yāni gṛhyante.' That is actions are Śrutilakṣaṇa and ācāralakṣaṇa. Samhya has several meanings as Amara says 'Samayāḥ śapathācārakālasiddhānta-saṃvidaḥ.'

+[^12]: हरदत्त on गौ. ध. सू. 19. 1 and गोविन्दराज on मनु 2.25 give the same fivefold classification. The Bhaviṣyapurāṇa (in Brāhmaparva 181, 10-15) sets out this fivefold dharma (Venk, Press ed.). The Gaut. Dh. 19. 1 says 'Ukto varṇadharmaścāśramadharmśca' and then proceeds to dilate upon prāyaścittas (i.e. naimittika dharma). The Āp. Dh. S. begins, 'atbātaḥ Sāmayācārikān _dharmān_ vyākhyāsyāmaḥ' (We shall expound the dharmas relating to the practices based on convention and adds immediately that the conventions settled by those who are conversant with dharma and the authorities and not any and every convention whatever,' The Āp. Gṛ. S. begins 'Atha karmāṇyācarād-yāni gṛhyante.' That is actions are Śrutilakṣaṇa and ācāralakṣaṇa. Samaya has several meanings as Amara says 'Samayāḥ śapathācārakālasiddhānta-saṁvidaḥ.'

No objection to using either ṃ or ṁ (IAST<https://en.wikipedia.org/wiki/IAST> vs ISO<https://en.wikipedia.org/wiki/ISO_15919>). परिवर्तने श्रमो मास्तु।

File: kANe/v1p1/01_meaning_of_dharma.md (hunk 1 above)
Comment (vvasuki) [reply kANe/v1p1/01_meaning_of_dharma.md:13]: स्तौषम् इति करोतु।  उपरि स्वरितचिह्नम् एव निष्कासितं दृश्यते।

File: kANe/v1p1/01_meaning_of_dharma.md (hunk 2 above)
Comment (vvasuki) [reply kANe/v1p1/01_meaning_of_dharma.md:15]: महोदये - उट्टङ्कयितुं न शक्नोति चेद् अपि पूर्वतनं पाठम् इतः अनुकृत्य (copy कृत्वा) लेप्तुं शक्नोतु। Lipika IME विषये ऽप्य् उक्तम्। तथा चास्योत्तरम् अत्रैवास्मिन् सूत्रे ददाति चेद् वरम्, यथा मया क्रियमाणम् - 
image

Comment (jaymahima) [reply kANe/v1p1/01_meaning_of_dharma.md:15]: इ॒मम॑ञ्ज॒स्पामु॒भये॑ अकृण्वत ध॒र्माण॑म॒ग्निं वि॒दथ॑स्य॒ साध॑नम् ।

Comment (jaymahima) [reply kANe/v1p1/01_meaning_of_dharma.md]: परिष्कृतम्।

File: kANe/v1p1/01_meaning_of_dharma.md (hunk 3 above)
Comment (jaymahima) [reply kANe/v1p1/01_meaning_of_dharma.md:30]: Screenshot 2022-11-24 at 6 25 39 PM

Comment (jaymahima) [reply kANe/v1p1/01_meaning_of_dharma.md:30]: Sorry mahodaya . 
Ok I understand your point -'No objection to using either ṃ or ṁ ([IAST](https://en.wikipedia.org/wiki/IAST) vs [ISO](https://en.wikipedia.org/wiki/ISO_15919)). परिवर्तने श्रमो मास्तु।'

**Pull request merged**